rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1043,8 +1041,13 @@ def persist(self) -> None:
 
             rasa.shared.utils.io.create_directory_for_file(tf_model_file)
 
-            if self.component_config[CHECKPOINT_MODEL]:
-                shutil.move(self.tmp_checkpoint_dir, model_path / "checkpoints")
+            if self.component_config[CHECKPOINT_MODEL] and self.tmp_checkpoint_dir:
+                self.model.load_weights(self.tmp_checkpoint_dir / "checkpoint.tf_model")
+                # For testing purposes only
```

Comment:
Could we change this comment to describe the purpose of the file more, I think it'll be unclear if we revisit this code why we're saving this file. Maybe something like "Save an empty file to flag that this model has been produced using checkpointing".

You could also use something like `touch` to do this (https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch) which might be a bit more elegant than pickling `None`